### PR TITLE
Setup yasg for openapi generators

### LINKED ISSUE #49: Fix Python to YAML API generator

### PR DESCRIPTION
**Pull request by avan1235**
closes #49 
Let's see the current effect on 
http://eaterslab.herokuapp.com/doc
http://eaterslab.herokuapp.com/api.yaml
And the current api still on http://eaterslab.herokuapp.com/api/beta/

### CODE REVIEW

File: django-server/server/server/urls.py [hunk 1]
```diff
@@ -15,10 +15,26 @@
...
+   public=True,
+   permission_classes=(permissions.AllowAny,),
+)
+
 urlpatterns = [
     path('', include('home.urls')),
     path('admin/', admin.site.urls),
     path('api/beta/', include('api.urls')),
+    # For documentation generation with drf_yasg
+    path(r'api.yaml', schema_view.without_ui(cache_timeout=0), name='schema-json'),
```
Comment (kantoniak) [inline django-server/server/server/urls.py]: Great job with the API! A few hints:
- URL should be `cafeterias`, these nouns are usually plural. Can you rename so we know how to do it with other views in future?
- Capacity is a private property, should not be displayed.
- Scheme is HTTP, but should be HTTPS
- There's currently no need to authenticate to get list of cafeterias, and even if we're not going to send passwords over HTTP.
- I think we don't have to support POST/PUT/PATCH/DELETE for cafeterias, we won't be implementing them on mobile right now.

I suggest enabling GET only over HTTPS with no authentication, this minimizes the work we have to do later.

Comment (kantoniak) [reply django-server/server/server/urls.py]: Regarding `api.yaml`, I think it's a good decision to keep it autogenerated for now. I will add missing camera methods in a follow-up PR and we can move to using generated API.

File: django-server/server/server/urls.py [hunk 2]
```diff
@@ -15,10 +15,26 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from drf_yasg.views import get_schema_view
+from drf_yasg import openapi
+from rest_framework import permissions
 
 
+# Code to generate beautiful API view
```
[the remark this answers is not in the document]
Comment (avan1235) [reply django-server/server/server/urls.py]: removed :(

File: django-server/server/server/urls.py (hunk 1 above)
Comment (avan1235) [reply django-server/server/server/urls.py]: cafeterias will be fixed on #50, these things can be made as a separated issue to deal with security of our app

**Comment by kantoniak** [comment]
@avan1235 HTTPS is still missing, right?

**Comment by avan1235** [comment]
Yeah. I was looking for some alternative to heroku but with no success. I think we can create some issue for this and try to fix it when model is ready

**Pull request merged**